src/error.ts

```diff
@@ -84,32 +86,16 @@ export class SessionNotFound extends ShopifyError {}
 export class CookieNotFound extends ShopifyError {}
 export class InvalidSession extends ShopifyError {}
 
-interface InvalidWebhookParams extends HttpResponseData {
+interface InvalidWebhookParams {
   message: string;
-  cause?: Error;
+  response: AdapterResponse;
 }
 export class InvalidWebhookError extends ShopifyError {
-  readonly response: HttpResponseData;
+  readonly response: AdapterResponse;
```

Comment:
I wonder if this typing would work for node - in that case `AdapterResponse` is `undefined`, so we wouldn't be surfacing any of the error information. I think we may want to keep this type the way it was before?

Reply:
Ah no that's fair - we want to provide the actual response in the object, but for node it's awkward because there'll be nothing there. Would it make sense to _also_ return a bit of information on the request to make it easier to debug errors?

Reply:
So, also include the original `AdapterRequest`?

Reply:
Something like that, though I'm not sure if there'll ever be any data there that's worth returning for webhooks. Maybe we should just keep what you did here and fix forward if we find any information we want to return going forward.